n2y/utils.py

```diff
@@ -99,5 +99,13 @@ def sanitize_filename(filename):
     return s
 
 
+def id_from_share_link(share_link):
+    hyphens_removed = share_link.replace("-", "")
```

Comment:
Perhaps we could just call `strip_dashes()` here?

Also it might make sense to rename `strip_dashes()` -> `strip_hyphens()` for consistency.